packages/api/src/utils/login.ts

```diff
@@ -228,9 +236,45 @@ export type SubscriptionInfo = {
   tenantId: string;
 };
 
+export declare type TokenRequest = {
+  scopes: Array<string>;
+  showDialog?: boolean;
+};
+export type LoginStatus = {
+  status: string;
+  token?: string;
+  accountInfo?: Record<string, unknown>;
+};
+export interface M365TokenProvider {
```

Comment:
Any document comment of this new interface, similar to the old `AppStudioTokenProvider`

Reply:
Thanks for the advice, already add.